.github/workflows/deploy.yml

```diff
@@ -91,7 +91,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: superfly/flyctl-actions/setup-flyctl@master
-      - run: flyctl deploy --config fly.staging.toml --remote-only -e GIT_SHA=$(git rev-parse HEAD)
+      - run: flyctl deploy --config fly.staging.toml --remote-only -e GIT_SHA=$(git rev-parse HEAD) -e ORIGIN="https://flow-docs-staging.fly.dev"
```

Comment:
Nice. This might be better than adding it via `fly secrets set`

Reply:
yeah, i like this a bit better because

- it's in code
- the value will be exposed when you run `flyctl config env -c fly.staging.toml` unlike secrets which are hashes